stout/flags.cc

```diff
@@ -21,32 +21,86 @@ namespace stout::flags {
 void Parser::AddAllOrExit(google::protobuf::Message* message) {
   const auto* descriptor = message->GetDescriptor();
 
+  auto exit_on_missing_name =
+      [](const google::protobuf::FieldDescriptor* field) {
+        std::cerr
+            << "Missing at least one flag name in 'names' for field '"
+            << field->full_name() << "'"
+            << std::endl;
+        std::exit(1);
+      };
+
+  auto exit_on_missing_help =
+      [](const google::protobuf::FieldDescriptor* field) {
+        std::cerr
+            << "Missing flag 'help' for field '"
+            << field->full_name() << "'"
+            << std::endl;
+        std::exit(1);
+      };
+
   for (int i = 0; i < descriptor->field_count(); i++) {
     const auto* field = descriptor->field(i);
 
-    const auto& flag = field->options().GetExtension(stout::v1::flag);
+    // We need this descriptor for the subcommand's logic.
+    const google::protobuf::OneofDescriptor* real_oneof_field =
```

Comment:
Since this isn't a "field" can we:
```suggestion
    const google::protobuf::OneofDescriptor* oneof =
```